Experiments/QKS/2019_03_05_qiskit_implementation/main.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+from sklearn.linear_model import LogisticRegression
+from qiskit import IBMQ
+import matplotlib.pyplot as plt
+
+from qks import QuantumKitchenSinks
+from make_frames import make_frames
+
+def logistic_regression(X_train, X_test, y_train, y_test):
+    lr = LogisticRegression(solver='lbfgs')
+    lr.fit(X_train, y_train)
+
+    train_acc = lr.score(X_train, y_train)
+    test_acc = lr.score(X_test, y_test)
+
+    print(
+        "accuracy\n----- \n training: {}\n test:     {}"
+          .format(train_acc, test_acc)
+         )
+
+    train_preds = lr.predict(X_train)
+    test_preds = lr.predict(X_test)
+
+    return train_preds, test_preds
+
+def make_plot(data, target, name, title):
+    plt.figure(figsize=(5, 5))
+    plt.title(title)
+    plt.scatter(data[:0], data[:1], s=5, c=target)
+    plt.savefig(name)
+
+    qc = make_qc(q, noisy)
+    p = X_train.shape[-1]
+
+    r = 1 if p / q < 1 else int(p / q)
+
+    QKS = QuantumKitchenSinks(qc, n_episodes=n_episodes, r=r, scale=scale, distribution='normal', n_trials=1000)
```

Comment:
This kind of logic shouldn't be inside `make_plot` function, rather in some separate function, e.g. `run_qks` or sth. like that.